src/nodeconductor_test/helpers.py

```diff
@@ -67,6 +67,7 @@ def go_to_main_page(driver):
 def make_screenshot(driver, name=None):
     if name is None:
         name = str(datetime.datetime.now()) + '.png'
+        name = name.replace(' ', '_')
```

Comment:
This code will be executed only if name is autogenerated. It is better to replace spaces if name was passed to `make_screenshot` explicitly too.